.github/renovate.json

```diff
@@ -0,0 +1,10 @@
+{
+    "enabledManagers": ["helm-requirements", "gradle-wrapper"],
```

Comment:
I would check the cnp-flux-config one instead for this repo, gradle-wrapper isn't relevant here,
https://github.com/hmcts/cnp-flux-config/blob/master/renovate.json#L2